### PR TITLE
Add bash script for extracting client files

### PR DESCRIPTION
**Pull request by Teppic1**
**Changes proposed:**

Simple bash script to automate extracting/copying client files on a Linux/Unix system.

It follows the instructions on the wiki but additionally copies camera files if found. It includes a few simple checks.

**Target branch(es):** 3.3.5

**Tests performed:** (Does it build, tested in-game, etc.)

I was able to extract and copy all files from my client.

### CODE REVIEW

**Comment by Faq** [comment]
# 22779

File: contrib/extractor.sh [hunk 1]
```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
+
+# This is where the WoW client files are located
+CLIENT_DIR=/mnt/disk/client
```
Comment (jackpoz) [inline contrib/extractor.sh]: I think it's better to ask for the paths in this script

Comment (Teppic1) [reply contrib/extractor.sh]: Updated.

File: contrib/extractor.sh [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+VMAP4ASSEMBLER=${SERVER_DIR}/bin/vmap4assembler
+MMAPS_GENERATOR=${SERVER_DIR}/bin/mmaps_generator
+
+check_files() {
+for i in $MAPEXTRACTOR $VMAP4EXTRACTOR $VMAP4ASSEMBLER $MMAPS_GENERATOR
+do
+	if [[ ! -x $i ]]
+	then
+		echo "Invalid executable: $i"
+		echo "Check tools are installed correctly"
```
Comment (illfated) [inline contrib/extractor.sh]: suggestion (for better understanding & readability in proper English):
```suggestion
		echo "Check that the extraction tools are installed correctly"
```

Comment (Teppic1) [reply contrib/extractor.sh]: This is normal shell script error language, it looks a bit weird if you put errors into full normal English :)

**Comment by jackpoz** [comment]
Thanks for the PR :)

**Pull request merged**